### PR TITLE
add active tasks to timeline

### PR DESCRIPTION
**Pull request by mattkrick**
**DO NOT MERGE UNTIL BASE BRANCH IS MASTER**

This is PR 5/5. It goes like this:
- PR 1: Suggested Actions
- PR 2: Remove welcome wizard
- PR 3: Timeline
- PR 4: what's new
- PR 5: **tasks shortlist**

TEST

- [ ] your active column tasks are shown in the timeline
- [ ] you can drag & drop them, but dragging & scrolling doesn't work too well yet (i'll need to build something for that, react-dnd-scrollzone only works in containers & this scroll is on the window)
- [ ] if you have 0 active tasks, a placeholder with a rally link appears

### CODE REVIEW

**Comment by mattkrick** [comment]
@jordanh ready for review

File: src/universal/components/TimelinePriorityTasks.tsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+  fontSize: ICON_SIZE.MD18,
+  verticalAlign: 'bottom'
+})
+
+const TaskList = styled('div')({
+  paddingTop: 32
+})
+
+class TimelinePriorityTasks extends Component<Props> {
+  getActiveTasks = memoize((tasks: TimelinePriorityTasks_viewer['tasks']) => {
```
Comment (jordanh) [inline src/universal/components/TimelinePriorityTasks.tsx]: This is a really cool pattern

**Comment by jordanh** [comment]
# Test Cycle 1

## Tests

- [ ] your active column tasks are shown in the timeline
- [ ] you can drag & drop them, but dragging & scrolling doesn't work too well yet (i'll need to build something for that, react-dnd-scrollzone only works in containers & this scroll is on the window)
- [ ] if you have 0 active tasks, a placeholder with a rally link appears

## Notes

### TS Warnings

```
WARNING in [at-loader] ./src/universal/components/TimelineFeedList.tsx:31:5
    TS2554: Expected 2-3 arguments, but got 1.

WARNING in [at-loader] ./src/universal/components/TimelineFeedList.tsx:67:3
    TS2345: Argument of type '{ direction: "forward"; getConnectionFromProps(props: Props): { readonly edges: ReadonlyArray<{ readonly cursor: any; readonly node: { readonly __typename: string; readonly id: string; readonly " $fragmentRefs": unique symbol; }; }>; readonly pageInfo: { ...; } | null; }; getFragmentVariables(prevVars: Variables, to...' is not assignable to parameter of type 'ConnectionConfig<Props>'.
  Types of property 'getConnectionFromProps' are incompatible.
    Type '(props: Props) => { readonly edges: ReadonlyArray<{ readonly cursor: any; readonly node: { readonly __typename: string; readonly id: string; readonly " $fragmentRefs": unique symbol; }; }>; readonly pageInfo: { ...; } | null; }' is not assignable to type '(props: Props) => ConnectionData | null | undefined'.
      Type '{ readonly edges: ReadonlyArray<{ readonly cursor: any; readonly node: { readonly __typename: string; readonly id: string; readonly " $fragmentRefs": unique symbol; }; }>; readonly pageInfo: { readonly hasNextPage: boolean; readonly endCursor: any; } | null; }' is not assignable to type 'ConnectionData'.
        Types of property 'pageInfo' are incompatible.
          Type '{ readonly hasNextPage: boolean; readonly endCursor: any; } | null' is not assignable to type 'Partial<PageInfo> | undefined'.
            Type 'null' is not assignable to type 'Partial<PageInfo> | undefined'.
```

**Comment by jordanh** [comment]
@mattkrick test cycle complete!

How difficult might it be to fix that scroll jump behavior?

**Comment by mattkrick** [comment]
not terribly difficult, but since it relies on context, i need to do it in a branch based off the one where we get rid of context.

**Comment by jordanh** [comment]
Let's do it! New branch, new life

**Comment by mattkrick** [comment]
ignoring the warnings for now. my PR to fix them was accepted, just waiting for the next version of the types to get published

**Pull request merged**